src/core.js

```diff
@@ -750,6 +750,12 @@ define([
 
         $modalBody.append($(find_usages({tableData: tableData})));
 
+        $modalBody.find('.usage-row').click(function() {
```

Comment:
How about making the question path into a link, instead of having the entire row be a click target? Would make it more obvious that clicking will make something happen.